### PR TITLE
{lib}[GCCcore/11.2.0] python-irodsclient v1.1.4 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by jefscheepers**
See #16299

### CODE REVIEW

File: easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+dependencies = [
+    ('Python', '3.9.6'),
+]
+
+use_pip = True
+# order is important!
+exts_list = [
+    ('defusedxml', '0.7.1', {}),
+    ('prettytable', '3.3.0', {}),
+    ('python-irodsclient', '1.1.4', {'modulename':'irods'})
```
Comment (boegel) [inline easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4.eb:17]: @jefscheepers Thanks a lot for the contribution!

You will need to add SHA256 checksums for each of the extensions here to make the CI tests pass (`eb --inject-checksums` is your friend here), and we would also appreciate from trivial style changes to make the easyconfig match the usual style in easyconfigs a bit better.

A good example to compare with is https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/g/GitPython/GitPython-3.1.24-GCCcore-11.2.0.eb

Do let us know if you need any help with this!

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by jefscheepers** [comment]
@boegel Is this better? 

Since I'll probably be making a GCCcore 10.3.0 & Python 3.9.5 version for our cluster, I added a Python version suffix as well (example taken from [here](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/g/GitPython/GitPython-3.1.9-GCCcore-9.3.0-Python-3.8.2.eb)). Is that okay, or should I remove it?

**Review by boegel — DISMISSED** [review]

File: easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4-GCCcore-11.2.0-Python-3.9.6.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonBundle'
+
+name = 'python-irodsclient'
+version = '1.1.4'
+versionsuffix = '-Python-%(pyver)s'
```
[the remark this answers is not in the document]
Comment (jefscheepers) [reply easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4-GCCcore-11.2.0-Python-3.9.6.eb:5]: Removed the versionsuffix and adapted the title

File: easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4-GCCcore-11.2.0-Python-3.9.6.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+homepage = 'https://github.com/irods/python-irodsclient'
+description = "A python API for iRODS"
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
```
Comment (smoors) [inline easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4-GCCcore-11.2.0-Python-3.9.6.eb:11]: @jefscheepers please add `binutils` builddep here?
```suggestion

builddependencies = [('binutils', '2.37')]

```

File: easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4-GCCcore-11.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonBundle'
+
+name = 'python-irodsclient'
+version = '1.1.4'
+
+homepage = 'https://github.com/irods/python-irodsclient'
+description = "A python API for iRODS"
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
```
Comment (smoors) [inline easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4-GCCcore-11.2.0.eb:10]: @jefscheepers please add `binutils` builddep here?
```suggestion

builddependencies = [('binutils', '2.37')]

```

File: easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4-GCCcore-11.2.0-Python-3.9.6.eb (hunk 2 above)
[the remark this answers is not in the document]
Comment (smoors) [reply easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4-GCCcore-11.2.0-Python-3.9.6.eb:5]: @jefscheepers please remove the duplicate easyconfig (i.e. the one with the Python versionsuffix)?

[the remark this answers is not in the document]
Comment (jefscheepers) [reply easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4-GCCcore-11.2.0-Python-3.9.6.eb]: Didn't see that one in my local, should be removed now

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node375.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/947fbe9b1db6b64257b5ed042107d4e7 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @jefscheepers!

**Pull request merged**